packages/api/endpoints/granules.js

```diff
@@ -1,7 +1,11 @@
+// @ts-check
```

Comment:
There actually wasn't anything too confusing to resolve, but that alone kind of gives me pause since we did have the PATCH changes in main. For the most part, it seems like those were able to be integrated in phase 3 seamlessly.

Reply:
https://github.com/nasa/cumulus/pull/3256/commits/eb6966f43e8e09b339bc303bd81e61cd3b489740